src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -853,7 +853,7 @@ public final void markDirty()
      */
     public void registerBlockPosition(@NotNull Block block, @NotNull final BlockPos pos)
     {
-        if (block instanceof BlockContainer)
+        if (block instanceof BlockContainer || block instanceof BlockMinecoloniesRack)
```

Comment:
Yeah that's what I meant, checking for the TileEntity capability would be the saver way to capture all kinds of chests